### PR TITLE
Update facility states to collate overdue patients data

### PR DESCRIPTION
**Pull request by PriyangaPKini**
**Story card:** [sc-XXXX](URL)

## Because

Enter the reason for raising this PR...

## This addresses

Enter the details of what all this covers...

## Test instructions

Enter detailed instructions for how to test this PR...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,24 @@ $ bin/setup
 
 If you encounter issues with this script, please open [a new issue with details](https://github.com/simpledotorg/simple-server/issues/new?title=Problems+with+bin/setup). Please include the entire log from bin/setup, as well as your computer / OS details.
 
-### Note for Apple Silicon M1 Macs
+### Setup Troubleshooting
+
+#### Setup fails when bundler tries to resolve `nokogiri`
```
Comment (danySam) [inline README.md:42]: Is this a common issue?

File: db/views/reporting_facility_states_v11.sql [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    AND monthly_follow_ups.facility_id = rf.facility_id
+LEFT OUTER JOIN monthly_diabetes_follow_ups
+    ON monthly_diabetes_follow_ups.month_date = cal.month_date
+    AND monthly_diabetes_follow_ups.facility_id = rf.facility_id
+LEFT OUTER JOIN reporting_facility_appointment_scheduled_days
+    ON reporting_facility_appointment_scheduled_days.month_date = cal.month_date
+    AND reporting_facility_appointment_scheduled_days.facility_id = rf.facility_id
+LEFT OUTER JOIN overdue_patients_aggregate
+    ON  overdue_patients_aggregate.month_date = cal.month_date
+    AND overdue_patients_aggregate.region_id = rf.facility_region_id
```
Comment (danySam) [inline db/views/reporting_facility_states_v11.sql:374]: Why do we use region_id on some joins and facility_id on others?

File: db/views/reporting_facility_states_v11.sql [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+LEFT OUTER JOIN monthly_follow_ups
+    ON monthly_follow_ups.month_date = cal.month_date
+    AND monthly_follow_ups.facility_id = rf.facility_id
+LEFT OUTER JOIN monthly_diabetes_follow_ups
+    ON monthly_diabetes_follow_ups.month_date = cal.month_date
+    AND monthly_diabetes_follow_ups.facility_id = rf.facility_id
+LEFT OUTER JOIN reporting_facility_appointment_scheduled_days
+    ON reporting_facility_appointment_scheduled_days.month_date = cal.month_date
+    AND reporting_facility_appointment_scheduled_days.facility_id = rf.facility_id
+LEFT OUTER JOIN overdue_patients_aggregate
```
Comment (danySam) [inline db/views/reporting_facility_states_v11.sql:372]: Can we not call this `monthly_overdue_patients`?

File: README.md (hunk 1 above)
Comment (PriyangaPKini) [reply README.md]: This is fixed when I rebased the branch

File: db/views/reporting_facility_states_v11.sql (hunk 3 above)
Comment (PriyangaPKini) [reply db/views/reporting_facility_states_v11.sql]: updated to `monthly_overdue_patients`

**Pull request closed without merging**